### PR TITLE
fix netdb bug

### PR DESCRIPTION
**Pull request by pillip8282**
1. if netdb calls exit() then tash will be destroyed. so I call 'return ' instead of calling exit()
2. usage is not correct. even though I turn off gethostbyaddr it shows address options

### CODE REVIEW

File: apps/system/netdb/netdb_main.c [hunk 1]
```diff
@@ -51,28 +51,28 @@
...
-   * @precondition	   : 1. Connect to Wi-Fi
-   *			   : 2. Get an IP address
-   *			   : 3. Run netdb
-   *			   :TASH> netdb --ipv4 127.0.0.1
-   * @postcondition	   :
-   */
+ * @testcase		   :gethostbyaddr
+ * @brief		   :To get hostname by IP address
+ * @scenario		   :
+ * @apicovered	   :
```
Comment (davidfather) [inline apps/system/netdb/netdb_main.c]: Please, align the colon.

File: apps/system/netdb/netdb_main.c [hunk 2]
```diff
@@ -51,28 +51,28 @@
...
-   * @precondition	   : 1. Connect to Wi-Fi
-   *			   : 2. Get an IP address
-   *			   : 3. Run netdb
-   *			   :TASH> netdb --host <URL>
-   * @postcondition	   :
-   */
+ * @testcase		   :dns_client
+ * @brief		   :To resolve hostname to IP address through DNS server
+ * @scenario		   :
+ * @apicovered	   :
```
Comment (davidfather) [inline apps/system/netdb/netdb_main.c]: Please, align the colon.

File: apps/system/netdb/netdb_main.c [hunk 3]
```diff
@@ -120,21 +120,20 @@
  * Private Functions
  ****************************************************************************/
 
-static void show_usage(FAR const char *progname, int exitcode) noreturn_function;
-static void show_usage(FAR const char *progname, int exitcode)
+static void show_usage(FAR const char *progname)
 {
-	fprintf(stderr, "USAGE: %s --ipv4 <ipv4-addr>\n", progname);
+
```
Comment (davidfather) [inline apps/system/netdb/netdb_main.c]: Please, remove this empty line.

**Comment by sunghan-chang** [comment]
@pillip8282 Please write details on commit message as you wrote in description of pull request.

```
1. if netdb calls exit() then tash will be destroyed. so I call 'return ' instead of calling exit()
2. usage is not correct. even though I turn off gethostbyaddr it shows address options
```

**Pull request merged**